python/py-parsing/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 name                py-parsing
 python.rootname     pyparsing
-version             3.0.5
+version             3.0.4
 revision            0
 epoch               1
```

Comment:
Decreasing the version requires incrementing the epoch.

I suggest adding a comment near here to inform others not to attempt updating the port to 3.0.5.

Reply:
@chrstphrchvz good point, thanks!

Reply:
> Decreasing the version requires incrementing the epoch.

The only possible exception I can think of for this is when the port always fails to install, which would not apply here.

Reply:
3.0.6 will likely be out before anyone re-attempts 3.0.5 in macports.